soc/riscv/riscv-privilege/telink_b91/CMakeLists.txt

```diff
@@ -12,4 +12,3 @@ zephyr_ld_options(-fuse-ld=bfd)
 
 # Set compile options
 zephyr_compile_options_ifdef(CONFIG_TELINK_B91_HWDSP -mext-dsp)
-zephyr_compile_options(-mno-relax)
```

Comment:
I propose do not delete it, but replace by 
``zephyr_compile_options_ifndef(CONFIG_RISCV_GP -mno-relax)``

Reply:
Done.